src/components/track-and-redirect/track-and-redirect-controller.ts

```diff
@@ -25,7 +28,10 @@ export const buildContactEmailServiceUrl = (req: Request): URL => {
         ]
       );
     } else {
-      logger.info(`Missing ${paramValue} in the request or session.`);
+      logger.info(
+        { trace: res.locals.sessionId },
```

Comment:
Why are we using the session ID as the trace here instead of the `res.locals.trace`?

Reply:
Changed to use trace from res.locals.